packages/dds/tree/src/feature-libraries/modular-schema/modularChangeFamily.ts

```diff
@@ -359,9 +364,12 @@ interface EncodedFieldChange {
 /**
  * @sealed
  */
-export class ModularEditBuilder extends ProgressiveEditBuilder<FieldChangeMap> {
+export class ModularEditBuilder
+    extends ProgressiveEditBuilderBase<FieldChangeMap>
+    implements ProgressiveEditBuilder<FieldChangeMap>
```

Comment:
Doesn't ` ProgressiveEditBuilderBase<FieldChangeMap>` already implement `ProgressiveEditBuilder<FieldChangeMap>`?

Reply:
It indeed did/does. This is just documentation (as with all "implements" in TS), but I agree it seems particularly verbose here.
I'm a bit of a purist when it comes to inheritance: I want the interface implementation to be completely independent of the inheritance part, and I want the inheritance part to be a private implementation detail. That last wish is not fulfilled with JS/TS classes, but I still end up trying to make the code answer the "what's your contract?" and "how do you meet that contract?" questions independently of one-another.
On a more practical note, if someone were to change `ProgressiveEditBuilderBase` so that it didn't implement `ProgressiveEditBuilder` anymore (which I'll admit seems pretty unlikely) then looking at `ModularEditBuilder` would no longer lead you to the `ProgressiveEditBuilder` that it aims to implement.